mitodl/mitxonline
# Documentation updates

#### What are the relevant tickets?
N/A

#### What's this PR do?
This PR updates documentation and configures github actions to publish them.

#### How should this be manually tested?
You can run `./pants docs ::` locally, then load the generated `dist/sphinx/index.html` to see the documentation. That should match the documentation deployed to pages here: https://mitodl.github.io/mitxonline/
